src/utils/git.d.ts

```diff
@@ -0,0 +1,371 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
💡☕️ Note\Question for archival reason: we borrowed the whole interface from : https://github.com/microsoft/vscode/tree/main/extensions/git , is there any way else we would have used this? or we have to duplicate this interface? Thanks heaps.

We should also document all this borrowed code between codebases please. It will help in maintenance and the historic reasons for anyone else scouting the code.

Reply:
This is required to be copied from the repository and this comment contains its location. I also believe its common knowledge in VS Code Extension development that these file must be copied to use the interface they offer, unless offered within a package manager.